File: app/src/main/java/cz/krystofcejchan/upocasi/fragments/SettingsFragment.java
package cz.krystofcejchan.upocasi.fragments;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import cz.krystofcejchan.upocasi.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey);
    }
}

File: app/src/main/java/cz/krystofcejchan/upocasi/fragments/SettingsFragment.kt
package cz.krystofcejchan.upocasi.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import cz.krystofcejchan.upocasi.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey)
    }
}